src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const result = {};
+  let styleRules = sourceString.split(';');
+
+  styleRules = styleRules.filter((rule) => rule.trim().length);
```

Comment:
[CODE KNOWLEDGE] - Don't need to make a copy before each operation. Moreover, methods like `filter` don't mutate the original array, so you can chain the operations directly without reassigning the `styleRules` variable.